spring-boot-project/spring-boot-docs/src/main/asciidoc/using-spring-boot.adoc

```diff
@@ -288,10 +287,11 @@ search dependencies by name. For example, with the appropriate Eclipse or STS pl
 installed, you can simply hit `ctrl-space` in the POM editor and type
 "`spring-boot-starter`" for a complete list.
 
-As explained in the <<spring-boot-features#boot-features-custom-starter,Creating your own starter>>
-section, third party starters should not start with `spring-boot` as it is reserved for
-official Spring Boot artifacts. A third-party starter for `acme` will be typically named
-`acme-spring-boot-starter`.
+As explained in the <<spring-boot-features#boot-features-custom-starter,Creating Your Own Starter>>
+section, third party starters should not start with `spring-boot`, as it is reserved for
+official Spring Boot artifacts. Rather, a third-party starter typically starts with the
+name of the organization that made it. For example, a third-party starter from `acme`
```

Comment:
It's not the name of the organisation but the name of the project. Perhaps "acme" isn't an ideal example name as it does suggest that it's an organisation rather than a project.

Reply:
OK. I changed "acme" to "thirdpartyproject". Wile E. Coyote will be disappointed. ;)